### ISSUE TITLE
Search within folder broken on 1.20.0

### ISSUE THREAD

**Issue by alexlamsl** [Microsoft/vscode#43196]
### Issue Type
Bug

### Description

- open a folder using VSCode
- goto Search tab (Ctrl+Shift+F)
- type string, press Enter

#### Expected
- display matches from all files within a folder

#### Actual
- only show matches from opened files

### VS Code Info

VS Code version: Code 1.20.0 (c63189deaa8e620f650cc28792b8f5f3363f2c5b, 2018-02-07T17:10:31.920Z)
OS version: Windows_NT ia32 10.0.14393

Extensions: none
Reproduces without extensions

**Comment by kevin-lozoya** [comment]
I have the same problem.

**Comment by smuuf** [comment]
Same here.

```
Version 1.20.0
Commit c63189deaa8e620f650cc28792b8f5f3363f2c5b
Date 2018-02-07T17:10:31.920Z
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architecture ia32
```

**Comment by roblourens** [comment]
Please do the following

- Run the command "Set log level" and change it to "Trace"
- Run the search again
- Show the search-related messages in the "Log (Window)" output panel

![image](https://user-images.githubusercontent.com/323878/35987940-02aae15a-0cb2-11e8-83f1-d95c5dc3881b.png)

**Comment by alexlamsl** [comment]
```
[2018-02-09 01:59:35.478] [renderer1] [trace] SearchService#search {"type":2,"folderQueries":[{"folder":{"$mid":1,"fsPath":"c:\\UglifyJS2","external":"file:///c%3A/UglifyJS2","path":"/c:/UglifyJS2","scheme":"file"},"excludePattern":{"**/.git":true,"**/.svn":true,"**/.hg":true,"**/CVS":true,"**/.DS_Store":true,"**/node_modules":true,"**/bower_components":true},"fileEncoding":"utf8"}],"usingSearchPaths":false,"maxResults":10000,"contentPattern":{"pattern":"join_vars","isRegExp":false,"isCaseSensitive":false,"isWordMatch":false,"wordSeparators":"`~!@#$%^&*()-=+[{]}\\|;:'\",.<>/?","isSmartCase":false},"useRipgrep":true,"disregardIgnoreFiles":false,"disregardExcludeSettings":false,"ignoreSymlinks":false}
[2018-02-09 01:59:35.487] [renderer1] [debug] SearchService#search rg --hidden --heading --line-number --color 'ansi' --colors 'path:none' --colors 'line:none' --colors 'match:fg:red' --colors 'match:style:nobold' --ignore-case -g '!/UglifyJS2/**/.git' -g '!/UglifyJS2/**/.svn' -g '!/UglifyJS2/**/.hg' -g '!/UglifyJS2/**/CVS' -g '!/UglifyJS2/**/.DS_Store' -g '!/UglifyJS2/**/node_modules' -g '!/UglifyJS2/**/bower_components' --max-filesize '314572800' --no-ignore-parent --follow --fixed-strings -- 'join_vars' 'c:\UglifyJS2'
 - cwd: c:/
 - Sibling clauses: {}
```

**Comment by roblourens** [comment]
Are any of the matching files covered by a gitignore file? You can test this by setting `"search.useIgnoreFiles": true`

**Comment by alexlamsl** [comment]
Is that a new feature? Because if not, I can assure this search would have returned a lot more results in any previous versions of VSCode.

**Comment by alexlamsl** [comment]
To answer your question more directly - I toggled that gear icon and no difference in the (limited) results returned.

**Comment by roblourens** [comment]
Not sure what the problem is. There are no errors in the "Log (window)" log right?

Does this happen for every folder you open? Every query?

**Comment by alexlamsl** [comment]
Yes - none of the folder search is working after 1.20

In fact, even if you open up the folder with previous tabs of opened files, they won't show up in search until I switch over to that tab.

**Comment by alexlamsl** [comment]
> Can you try invoking rg.exe directly from the command prompt?

Using the built-in console window (powershell) and run:
```
& "C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\vscode-ripgrep\bin\rg.exe" join_vars
```
... gave me nothing in return.

If I run it under Command Prompt:
```
"C:\Program Files (x86)\Microsoft VS Code\resources\app\node_modules\vscode-ripgrep\bin\rg.exe" join_vars
```
I get a pop-up saying something like "The Program can't start because VCRUNTIME140.dll ..."

**Comment by roblourens** [comment]
Oh, it must not be statically compiled properly.

Can you check if the Insiders version works?

**Comment by chramc** [comment]
I have the same problem.

**Comment by alexlamsl** [comment]
@roblourens I can't switch to Insiders for now, but I can confirm that x64 is not affected - works for both 1.19.3 and 1.20.0

**Comment by roblourens** [comment]
I have uploaded a fixed version of this binary, version 0.7.1-patch.0. No changes required in vscode.
